source/reference/color_schemes.rst

```diff
@@ -0,0 +1,256 @@
+to change the color scheme
+depending on the file type, time of day, etc.
+
+
+Structure of a Color Scheme File
+================================
+
+Color scheme files are based
+on the Property List format.
+All color scheme files share
+the same top-level structure.
+
+.. insert top-level example here
+
+``name``
+   Optional.
+   Name of the color scheme.
+   Ignored by Sublime Text.
+
+``settings``
+   Array of dict elements.
+   See :ref:`Subelements of Settings` for more information.
+
+``uuid``
+   Optional.
+   A unique identifier for the file. Ignored by Sublime Text.
+
+
+Subelements of Settings
+***********************
+
+Sublime Text supports
+the following color scheme settings:
+
+
+Global Settings
+---------------
+
+Not associated with any scope.
+These settings affect global visual items in the editing area.
+
+``activeGuide``
+   Color of the guide lined up with the caret.
+   Only used if the ``indent_guide_options`` setting
+   is set to ``draw_active``.
+
+``background``
+   Backgound color of the view.
+
+``bracketContentsForeground``
+   Color of bracketed sections of text
+   when the caret is in a bracket section.
+   Only used when the ``match_brackets`` setting
+   is set to `true`.
+
+``bracketContentsOptions``
+   XXX when the caret is in a bracket section.
+   Only used when the ``match_brackets`` setting
+   is set to ``true``.
+
+   Options: ``underline``, ``stippled_underline``, ``squiggly_underline``.
+   `underline` indicates the text should be drawn
+   using the given color, not just the underline.
+
+``bracketsForeground``
+   Foreground color of the brackets
+   when the caret is next to a bracket.
+   Only used when the ``match_brackets`` setting
+   is set to ``true``.
+
+``bracketsForeground``
+   Background color of the brackets
+   when the caret is next to a bracket.
+   Only used when the ``match_brackets`` setting
+   is set to ``true``.
+
+``bracketsOptions``
+   XXX
+
+   Options: ``underline``, ``stippled_underline``, ``squiggly_underline``.
+   ``underline`` indicates the text should be drawn
+   using the given color, not just the underline.
+
+``caret``
+   Color of the caret.
+
+``findHighlight``
+   Background color of regions matching the current search.
+
+``findHighlightForeground``
+   Background color of regions matching the current search.
+
+``foreground``
+   Foreground color for the view.
+
+``guide``
+   Color of the guides displayed to mark nesting levels.
+
+``gutter``
+   Background color of the gutter.
+
+``gutterForeground``
+   Foreground color of the gutter.
+
+``inactiveSelection``
+   Color of inactive selections (inactive view).
+
+``invisibles``
+  Ignored by Sublime Text.
+
+``lineHihglight``
+   Color of the line the caret is in.
+   Only used when the ``higlight_line`` setting is set to ``true``.
+
+``selection``
+   Color of the selection regions.
+
+``selectionBackground``
+   Background color of the selection regions.
```

Comment:
we could screenshots for all of these and doublecheck when we do. in other words, won't check now.